qan-api2/my.cnf

```diff
@@ -0,0 +1,6 @@
+[mysqld]
```

Comment:
TBH, I have no idea why we have these two files in QAN repository, because we don't use MySQL in QAN.

Reply:
Removed both